packages/medusa/src/services/line-item.ts

```diff
@@ -378,6 +378,31 @@ class LineItemService extends BaseService {
     )
   }
 
+  /**
+   * Deletes a line item with the tax lines.
+   * @param {string} id - the id of the line item to delete
+   * @return {Promise<LineItem | undefined>} the result of the delete operation
+   */
+  async deleteWithTaxLines(id: string): Promise<LineItem | undefined> {
```

Comment:
thought: would it make sense to remove this method and make tax line deletion part of the regular `lineItemService.delete` method as we would otherwise be creating orphaned tax lines?

Reply:
Not sure it should be part of that pr as it could have other implication in other places no? If there is no other side effects then it is a good idea to do it now :)

Reply:
consider it non-blocking then :)